### ISSUE TITLE
Cuda not available after isntalling the environment

### ISSUE THREAD

**Issue by CristopherUbotica** [ESA-PhiLab/PyRawS#28]
Hello,

I installed the environment from the .yaml file following the procedure described in the readme, and in my case cuda was not available. Is it problem of the environment? Or am I missing something?

Thank you in advance for your time and consideration.

Kind regards,

Cristopher

**Comment by sirbastiano** [comment]
Hi Cris,
could you detail the OS, and the steps you have done so we can replicate your bug?

**Comment by CristopherUbotica** [comment]
Hi Roberto,

I am using Ubuntu 22.04. I run the the 'pyraws_install.sh' shell script to set up the environment.  After that, I activated the environment and tried torch.cuda.is_available() on Python. 

Can the problem be that the cuda version that I have installed in my OS is not compatible with the environment version? Or should the environment be enough to run cuda in pytorch? In that case, the problem wouldn't be part of the repo.

Thank you in advance!
